syftlib/src/main/java/org/openmined/syft/Job.kt

```diff
@@ -0,0 +1,45 @@
+package org.openmined.syft
+
+
+import io.reactivex.Flowable
+import io.reactivex.Single
+import io.reactivex.processors.PublishProcessor
+import kotlinx.serialization.Serializable
+import org.openmined.syft.network.NetworkMessage
+import org.openmined.syft.threading.ProcessSchedulers
+
+@Serializable
+class Job(val modelName: String, val version: String) {
```

Comment:
`Job` is a class in Kotlin already. I was wondering if we could use another name for this class.
(just wondering)

Reply:
Surely seems no problem